api/machine/machine.proto

```diff
@@ -262,3 +263,7 @@ message LogsRequest {
   // driver might be default "containerd" or "cri"
   common.ContainerDriver driver = 3;
 }
+
+message ReadRequest {
+  string path = 1;
```

Comment:
Not for the first-pass feature, but properties of `Offset` and `Limit` would be useful from an API perspective.

Reply:
Agree. Was debating if this should actually be a stream but yeah we can take a closer look in the future.

Reply:
+1 to streaming _receive_, but this is the request side, anyway.  I suppose there is an argument to offering flow control via a streaming request side interface, but that's even further out, I think.